test/input_gen/genInput.py

```diff
@@ -1,24 +1,38 @@
 #!/usr/bin/env python3
 ##
 # Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
+# Copyright (C) 2020 Parichay Kapoor <pk.kapoor@samsung.com>
 #
 # SPDX-License-Identifier: Apache-2.0-only
 #
 # @file genTestInput.py
 # @brief Generate test input
 # @author Jijoong Moon <jijoong.moon@samsung.com>
+# @author Parichay Kapoor <pk.kapoor@samsung.com>
 
 import sys
 import os
 import numpy as np
 import tensorflow as tf
 from tensorflow.python.framework import dtypes
 import struct
+import random
+
+DEBUG = True
+
+# Fix the seeds across frameworks
+SEED = 1234
+tf.compat.v1.reset_default_graph()
+os.environ['TF_CUDNN_DETERMINISTIC'] = '1'
+os.environ['PYTHONHASSEED'] = str(SEED)
```

Comment:
I guess this does not have any effect as [environment variable is processed in launch process](https://stackoverflow.com/questions/25684349/can-os-environpythonhashseed-be-set-dynamically-from-within-an-application).

Reply:
Yes, it's true. 
Removed setting `PYTHONHASHSEED` environment variable setting.
Moved `TF_CUDNN_DETERMINISTIC` environment variable before the TensorFlow is imported.